DataImport.Web/Startup.cs

```diff
@@ -82,7 +88,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMediatR(typeof(Startup));
             services.AddHttpContextAccessor();
 
+#if DEBUG
+            services.AddHttpClient("ServerName").ConfigurePrimaryHttpMessageHandler(_ => new HttpClientHandler
```

Comment:
Use a different name like _LocalHttpClient_ or something similar to that, and if it is possible save it as constant in a common class in order to use it in the different spots of the code which is required.